### PR TITLE
(#22) Added support for checkupdates and apt_checkupdates

### PR DESCRIPTION
**Pull request by KoenDierckx**

### CODE REVIEW

File: application/package.rb [hunk 1]
```diff
@@ -108,6 +110,9 @@ def main
                   if ['count', 'md5'].include?(configuration[:action])
                     status = "%s" % [result[:data][:output]]
                     puts(pattern % [result[:sender], status])
+                  elsif ['yum_checkupdates', 'apt_update', 'checkupdates', 'apt_checkupdates'].include?(configuration[:action])
+                    status = result[:data][:outdated_packages].map{ |package| package[:package] }.join(' ')
+                    puts(pattern % [result[:sender], status])
```
Comment (ripienaar) [inline application/package.rb]: why the 2 different formats? I think this is quite nice rather than the format above?

Comment (KoenDierckx) [reply application/package.rb]: I tough having more information is good, but perhaps this went a bit to far with returning all information. Updated code to return name + version in verbose mode

File: application/package.rb [hunk 2]
```diff
@@ -91,10 +91,13 @@ def main
           pkg_result.each do |result|
             if result[:statuscode] == 0
               if pkg.verbose
-                if ['count', 'md5'].include?(configuration[:action])
-                  puts(pattern % [result[:sender], result[:data][:output]])
-                else
+                if configuration[:action] == 'status'
                   puts(pattern % [result[:sender], result[:data][:ensure]])
+                elsif ['yum_checkupdates', 'apt_update', 'checkupdates', 'apt_checkupdates'].include?(configuration[:action])
+                  status = result[:data][:outdated_packages].map{ |package| "%s-%s" % package[:package], package[:version] }.join(' ')
```
Comment (ripienaar) [inline application/package.rb]: tests failing here, should be `"%s-%s" % [foo, bar]`

**Comment by KoenDierckx** [comment]
I fixed the tests
But merging the commits has been more complicated then expected. 
Git is like a swiss army knife...I keep cutting myself while playing around with it ;)

**Comment by ripienaar** [comment]
Thanks :) I'll see if I can squash it for you, thanks a lot, will review soon

**Pull request merged**